docker/centos7/entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+pbs_conf_file=/etc/pbs.conf
+mom_conf_file=/var/spool/pbs/mom_priv/config
+hostname=`hostname`
+
+# replace hostname in pbs.conf and mom_priv/config
+sed -i "s/PBS_SERVER=.*/PBS_SERVER=$hostname/" $pbs_conf_file
+sed -i "s/\$clienthost .*/\$clienthost $hostname/" $mom_conf_file
+
+# do not start server if PBS_START_SERVER is 0
+# by default PBS_START_SERVER is 1
```

Comment:
Only true if /opt/pbs/sbin/pbs_server.bin exists. Take a look at pbs_postinstall. Much of what you're doing here is also done in the postinstall script.

Will you need to accommodate execution and client installs as well?

Reply:
The docker image has `pbspro_server.rpm` installed so `pbs_server.bin` exists. The `pbs_postinstall` file creates the `pbs.conf` file after installation, which is done during `docker build`. This script runs when a new container is launched using `docker run`.

Reply:
The other option is to change the init script to do this automatically.  All other pbs.conf variables can be overridden by an environmental variable of the same name.  The START variables are special since pbs_load_conf() is not handling them, the init script is.  We can change the init script to follow along with the rest of the pbs.conf variables.